README.md

```diff
@@ -14,7 +14,7 @@ All of the Adafruit board definitions in the sketch through the library manager.
 
 The bsec library needs to be at (1.4.703).
 
-As well the Adafruit Bluefruit nrf51 library through the library manager.
+As well as the Adafruit Bluefruit nrf51 library through the library manager.
```

Comment:
(copying question from @illperipherals to here) Should this be nrf52 ?

Reply:
Just how it is in the Arduino libraries